Formula/molten-vk.rb

```diff
@@ -2,55 +2,54 @@ class MoltenVk < Formula
   desc "Implementation of the Vulkan graphics and compute API on top of Metal"
   homepage "https://github.com/KhronosGroup/MoltenVK"
   license "Apache-2.0"
-  revision 1
 
   stable do
-    url "https://github.com/KhronosGroup/MoltenVK/archive/v1.1.10.tar.gz"
-    sha256 "fac11c2501195c9ce042103685c7778e35484562e6c084963a22072dd0a602e0"
+    url "https://github.com/KhronosGroup/MoltenVK/archive/v1.1.11.tar.gz"
+    sha256 "938ea0ba13c6538b0ee505ab391a3020f206ab9d29c869f20dd19318a4ee8997"
 
     # MoltenVK depends on very specific revisions of its dependencies.
     # For each resource the path to the file describing the expected
     # revision is listed.
-    resource "cereal" do
-      # ExternalRevisions/cereal_repo_revision
-      url "https://github.com/USCiLab/cereal.git",
-          revision: "51cbda5f30e56c801c07fe3d3aba5d7fb9e6cca4"
+    resource "SPIRV-Cross" do
+      # ExternalRevisions/SPIRV-Cross_repo_revision
+      url "https://github.com/KhronosGroup/SPIRV-Cross.git",
+          revision: "61c603f3baa5270e04bcfb6acf83c654e3c57679"
```

Comment:
just curious, why is there a resource order change in here? Thanks!

Reply:
With this change, all resources are in the order they're listed in the repositories the revisions come from:
https://github.com/KhronosGroup/MoltenVK/tree/v1.1.11/ExternalRevisions
https://github.com/KhronosGroup/glslang/blob/73c9630da979017b2f7e19c6549e2bdb93d9b238/known_good.json

This makes it easier to just go down the list and grab the new revisions.

Reply:
make sense, thanks!